packages/vitaminjs-build/bin/vitamin-start.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+const program = require('commander');
+const start = require('../cli/start');
+
+program
+    .name('start')
+    .alias('s')
+    .option('--no-hmr', 'Disable hot reload')
```

Comment:
Should not this be `--no-hot` here. Or `--hmr` in vitamin-build option? To avoid different wording for the same thing if I'm right

Reply:
oh it was already like that, my bad x)